services/gem/gem.js

```diff
@@ -94,10 +87,7 @@ class GemDownloads extends BaseService {
     version = (version === "stable") ? version : semver.valid(version);
     const label = this._getLabel(version, info);
     const apiUrl = this._getApiUrl(repo, info);
-    const json = await this._sendAndCacheRequest(apiUrl, {
-      headers: { 'Accept': 'application/atom+json,application/json' }
```

Comment:
This particular line had a slightly different `Accept` header, any idea why?

Reply:
It was added in https://github.com/badges/shields/pull/212 but there is no explanation there. It works fine with the standard `Accept: application/json` it so I didn't see the need to cargo cult it.